tests/product/base_product_case.py

```diff
@@ -335,16 +335,23 @@ def assert_installed(self, container, cluster=None, msg=None):
             cluster = self.cluster
         try:
             check_rpm = cluster.exec_cmd_on_host(
-                container, 'rpm -q presto')
-            self.assertEqual(self.presto_rpm_filename[:-4] + '\n', check_rpm,
-                             msg=msg)
+                container, 'rpm -q presto-server-rpm')
+            self.assertRegexpMatches(
+                check_rpm, self.presto_rpm_filename[:-4] + r'.*\n', msg=msg
```

Comment:
there are extra things (e.g. -0:0.113.SNAPSHOT-1.x86_64 or something) which might change which we don't care about